### PR TITLE
[WIP] Create security-policy.md

### PR DESCRIPTION
**Pull request by jhutchings1**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/master/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

-----

Related to #981, I have created a first draft of documentation which describes best practices for a security policy.

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
@jhutchings1 Looks good to me! When CI is green and you're ready for this to 🚢 give me a shout and I'll merge it in.

**Comment by siguientepaso** [comment]
Cres que valla a funcionar??

**Comment by mlinksva** [comment]
I feel like `security-policy` is a more narrowly scoped than other guides. `security` on the other hand, would fill a big omission from the guides. Basically expand and reframe around keeping your project and its users secure. Having a security policy would be part of that guide-ance. Just a thought, feel free to disregard!

**Comment by EdOverflow** [comment]
I like @mlinksva's idea. It would be nice to have a general security section in the https://opensource.guide/ that covers best practices and helps developers secure their GitHub projects.

File: _articles/security-policy.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+lang: en
+title: Security policy
+description: Help users report security issues by providing a security policy
+class: best-practices
+image: /assets/images/cards/best-practices.png
+---
+
+## Why do I need a security policy? 
+
+Even small projects in open-source can have a big impact because of the way ideas can move through the community. When it comes to software security, vulnerabilities can move through the community just as quickly as great ideas, so it's important that every open-source project think about their security policy. A security policy helps your community by a) letting them know what you will support in terms of security issues, and b) letting them know how to work with you when they discover security issues.
```
Comment (rishu2403) [inline _articles/security-policy.md]: 'the way ideas can move through the community.' No need to add 'can' again.

**Comment by LalitoConstantino** [comment]
> * [x]  Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/master/CONTRIBUTING.md)?
> * [x]  Have you explained what your changes do, and why they add value to the Guides?
> 
> Related to #981, I have created a first draft of documentation which describes best practices for a security policy.

**Comment by jhutchings1** [comment]
Thanks all for the ✅ . I haven’t had the cycles to build this into the TOC properly, but I like the idea of a general security section as well. I’m catching up from vacation and GitHub summit, but will revisit this soon. If anyone wants to run with it in the interim, I’m 👍

**Comment by LalitoConstantino** [comment]
Solo es la guia...

La aplicacion solo la tiene un sistema...

**Comment by boohoo555** [comment]
git@github.com:nodegui/react-nodegui.git

**Comment by EdOverflow** [comment]
Just noticed you might want to consider revising the following passage under "How to Contribute to Open Source" since, in my opinion, this contradicts the advice in this pull request.

https://github.com/github/opensource.guide/blob/681597c6ec5181e953df9977a18e7b4f8db365b4/_articles/how-to-contribute.md#L418

**Comment by mlinksva** [comment]
@EdOverflow I don't think it does:

> unless you need to share sensitive information (such as a security issue or serious conduct violation)

Am I missing something?

**Comment by EdOverflow** [comment]
You are right, @mlinksva. I completely misread that. Thank you for pointing that out. :+1:

**Comment by jhutchings1** [comment]
Is there somebody that works on this project that can help out with illustrations for this security page? That's a bit outside my skillset 😄

**Comment by jhutchings1** [comment]
Also, the build is breaking as a result of not having localized versions of my new article. How do we get those created? Thanks!

**Comment by mlinksva** [comment]
@jhutchings1 might need to add something to tests to allow for not-yet-translated articles. I don't think any articles have been added since the original set so we haven't had the issue yet. Or maybe the right thing is to add stubs for to each translation for a new article asking for translations? Either would seem reasonable to me.

I believe @sophshep was the initial site designer and might be able to point you somewhere, maybe inside GitHub, regarding illustrations.

**Comment by sophshep** [comment]
> Is there somebody that works on this project that can help out with illustrations for this security page?
> I believe @sophshep was the initial site designer and might be able to point you somewhere, maybe inside GitHub, regarding illustrations.

Yep, I was! These illustrations were done by @jeejkang who is no longer at GitHub, but @tonyjaramillo (manager of our art/illustration team) may be able to help.

**Comment by jhutchings1** [comment]
I haven't had the cycles to get this over the line, so happy to leave this closed for now.

**Comment by suusha1340** [comment]
From 7f77bd09c252888a58bcbdcc2424acf81d32e130 Mon Sep 17 00:00:00 2001
From: Justin Hutchings <jhutchings1@users.noreply.github.com>
Date: Mon, 3 Jun 2019 16:13:57 -0700
Subject: [PATCH 1/5] Create security-policy.md

---
 _articles/security-policy.md | 32 ++++++++++++++++++++++++++++++++
 1 file changed, 32 insertions(+)
 create mode 100644 _articles/security-policy.md

diff --git a/_articles/security-policy.md b/_articles/security-policy.md
new file mode 100644
index 0000000000..7d93d54a61
--- /dev/null
+++ b/_articles/security-policy.md
@@ -0,0 +1,32 @@
+---
+lang: en
+title: Security policy
+description: Help users report security issues by providing a security policy
+class: best-practices
+image: /assets/images/cards/best-practices.png
+---
+
+## Why do I need a security policy? 
+
+Even small projects in open-source can have a big impact because of the way ideas can move through the community. When it comes to software security, vulnerabilities can move through the community just as quickly as great ideas, so it's important that every open-source project think about their security policy. A security policy helps your community by a) letting them know what you will support in terms of security issues, and b) letting them know how to work with you when they discover security issues.  
+
+## Components of a good security policy
+
+### Scope/Supported versions
+
+Tell users about the parts of your project which are within scope of the security policy. For example, if you will not accept security bugs on test collateral, or samples, you should describe those limitations here. If your project has any official releases or versions, you should let users know which ones are being officially supported here. 
+
+### Disclosure policy 
+
+In the disclosure policy section, describe how and where users should disclose security vulnerabilities here. This is the right place for a link to the private inbox you might host, a link to your HackerOne, or a link to your own custom website. 
+
+If you have any security requirements on the communications (eg, no attachments, signed with a PGP key, etc), you should also list them here. 
+
+### Safe harbor 
+
+Safe harbor provisions are important in terms of setting boundaries as to what constitutes ethical hacking, vs unlawful hacking on your project. These sections are important, so if your company or project has lawyers, now's the time to call them. A good security policy should encourage security researchers to report security issues responsibly, and a good safe harbor provision gives those researches confidence that they won't be prosecuted for doing so. The [GitHub Bug Bounty Program](https://github.blog/2019-02-19-five-years-of-the-github-bug-bounty-program/#legal-safe-harbor) provides more information about how GitHub uses a safe harbor provision to protect hackers provided that they abide by certain limitations. 
+
+## See also
+* [HackerOne's security policy builder](https://hackerone.com/policy-builder/)
+* [Securitytxt.org](https://securitytxt.org)
+* [NTIA Coordinated Vulnerability Disclosure template] (https://www.ntia.doc.gov/files/ntia/publications/ntia_vuln_disclosure_early_stage_template.pdf)

From 4f2bd1500f86b0851afb55a6c2015129ca214d75 Mon Sep 17 00:00:00 2001
From: Justin Hutchings <jhutchings1@users.noreply.github.com>
Date: Thu, 22 Aug 2019 17:19:01 -0700
Subject: [PATCH 2/5] Update security-policy.md

Incorporating feedback
---
 _articles/security-policy.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/_articles/security-policy.md b/_articles/security-policy.md
index 7d93d54a61..5a8005eb22 100644
--- a/_articles/security-policy.md
+++ b/_articles/security-policy.md
@@ -8,7 +8,7 @@ image: /assets/images/cards/best-practices.png
 
 ## Why do I need a security policy? 
 
-Even small projects in open-source can have a big impact because of the way ideas can move through the community. When it comes to software security, vulnerabilities can move through the community just as quickly as great ideas, so it's important that every open-source project think about their security policy. A security policy helps your community by a) letting them know what you will support in terms of security issues, and b) letting them know how to work with you when they discover security issues.  
+Even small projects in open-source can have a big impact because of the way ideas move through the community. When it comes to software security, vulnerabilities can move through the community just as quickly as great ideas, so it's important that every open-source project think about their security policy. A security policy helps your community by a) letting them know what you will support in terms of security issues, and b) letting them know how to work with you when they discover security issues.  
 
 ## Components of a good security policy
 

From 68085fd59071360ab5169da126be5d9c5317dafd Mon Sep 17 00:00:00 2001
From: Justin Hutchings <jhutchings1@users.noreply.github.com>
Date: Thu, 22 Aug 2019 17:52:37 -0700
Subject: [PATCH 3/5] Broadening the security section

---
 _articles/security-policy.md      | 32 ------------------
 _articles/think-about-security.md | 54 +++++++++++++++++++++++++++++++
 2 files changed, 54 insertions(+), 32 deletions(-)
 delete mode 100644 _articles/security-policy.md
 create mode 100644 _articles/think-about-security.md

diff --git a/_articles/security-policy.md b/_articles/security-policy.md
deleted file mode 100644
index 5a8005eb22..0000000000
--- a/_articles/security-policy.md
+++ /dev/null
@@ -1,32 +0,0 @@
----
-lang: en
-title: Security policy
-description: Help users report security issues by providing a security policy
-class: best-practices
-image: /assets/images/cards/best-practices.png
----
-
-## Why do I need a security policy? 
-
-Even small projects in open-source can have a big impact because of the way ideas move through the community. When it comes to software security, vulnerabilities can move through the community just as quickly as great ideas, so it's important that every open-source project think about their security policy. A security policy helps your community by a) letting them know what you will support in terms of security issues, and b) letting them know how to work with you when they discover security issues.  
-
-## Components of a good security policy
-
-### Scope/Supported versions
-
-Tell users about the parts of your project which are within scope of the security policy. For example, if you will not accept security bugs on test collateral, or samples, you should describe those limitations here. If your project has any official releases or versions, you should let users know which ones are being officially supported here. 
-
-### Disclosure policy 
-
-In the disclosure policy section, describe how and where users should disclose security vulnerabilities here. This is the right place for a link to the private inbox you might host, a link to your HackerOne, or a link to your own custom website. 
-
-If you have any security requirements on the communications (eg, no attachments, signed with a PGP key, etc), you should also list them here. 
-
-### Safe harbor 
-
-Safe harbor provisions are important in terms of setting boundaries as to what constitutes ethical hacking, vs unlawful hacking on your project. These sections are important, so if your company or project has lawyers, now's the time to call them. A good security policy should encourage security researchers to report security issues responsibly, and a good safe harbor provision gives those researches confidence that they won't be prosecuted for doing so. The [GitHub Bug Bounty Program](https://github.blog/2019-02-19-five-years-of-the-github-bug-bounty-program/#legal-safe-harbor) provides more information about how GitHub uses a safe harbor provision to protect hackers provided that they abide by certain limitations. 
-
-## See also
-* [HackerOne's security policy builder](https://hackerone.com/policy-builder/)
-* [Securitytxt.org](https://securitytxt.org)
-* [NTIA Coordinated Vulnerability Disclosure template] (https://www.ntia.doc.gov/files/ntia/publications/ntia_vuln_disclosure_early_stage_template.pdf)
diff --git a/_articles/think-about-security.md b/_articles/think-about-security.md
new file mode 100644
index 0000000000..f51fa74579
--- /dev/null
+++ b/_articles/think-about-security.md
@@ -0,0 +1,54 @@
+---
+lang: en
+title: Think about security
+description: Build your open source project with security in mind
+class: best-practices
+image: /assets/images/cards/best-practices.png
+---
+
+## Why is security such a big deal? 
+
+Even small projects in open-source can have a big impact because of the way ideas move through the community. When it comes to software security, vulnerabilities can move through the community just as quickly as great ideas, so it's important that every open-source project think about how they will approach security. 
+
+## Secure your software supply chain
+
+Software component analysis is an important part of protecting your software supply chain. When open-source modules that you depend on have vulnerabilities, your users are at risk! Services like [GitHub](https://github.com/), [Snyk](https://snyk.io/), or [WhiteSource](https://www.whitesourcesoftware.com/) offer vulnerability scanning for free for open-source projects. These services will send you security alerts when your projects have a vulnerability and will include information on the impact of the vulnerability.
+
+Docker containers are another major source of vulnerabilities. Any of the code that's part of a Docker container that you run could have vulnerabilities. The open-source project [Clair](https://github.com/coreos/clair) does scanning of Docker containers for known vulnerabilities. There are also many commercial security vendors which may perform more comprehensive scanning of Docker containers. 
+
+## Automate repetitive tasks
+
+Updating dependencies can be a frequent, and repetitive task. Many services offer free dependency updates including [Dependabot](https://dependabot.com/) (which is built-into GitHub as a beta right now), and [GreenKeeper](https://greenkeeper.io/). These tools will automatically generate pull requests with the required version updates in response to security vulnerabilities, and dramatically improve the experience of updating dependencies. 
+
+## Create a security policy
+
+ A security policy helps your community by a) letting them know what you will support in terms of security issues, and b) letting them know how to work with you when they discover security issues.  
+
+### Scope/Supported versions
+
+Tell users about the parts of your project which are within scope of the security policy. For example, if you will not accept security bugs on test collateral, or samples, you should describe those limitations here. If your project has any official releases or versions, you should let users know which ones are being officially supported here. 
+
+### Disclosure policy 
+
+In the disclosure policy section, describe how and where users should disclose security vulnerabilities here. This is the right place for a link to the private inbox you might host, a link to your HackerOne, or a link to your own custom website. 
+
+If you have any security requirements on the communications (eg, no attachments, signed with a PGP key, etc), you should also list them here. 
+
+### Safe harbor 
+
+Safe harbor provisions are important in terms of setting boundaries as to what constitutes ethical hacking, vs unlawful hacking on your project. These sections are important, so if your company or project has lawyers, now's the time to call them. A good security policy should encourage security researchers to report security issues responsibly, and a good safe harbor provision gives those researches confidence that they won't be prosecuted for doing so. The [GitHub Bug Bounty Program](https://github.blog/2019-02-19-five-years-of-the-github-bug-bounty-program/#legal-safe-harbor) provides more information about how GitHub uses a safe harbor provision to protect hackers provided that they abide by certain limitations. 
+
+## Responsibly disclose your vulnerabilities
+
+When you learn about a security vulnerability in your code, it's important that you fix and disclose that issue responsibly. In open-source projects, this can be challenging, since everything is open by default. 
+
+GitHub offers a beta feature called [maintainer security advisories](https://help.github.com/en/articles/about-maintainer-security-advisories) which allow you to discuss, fix, and disclose vulnerabilities without disclosing them before they are ready. Some projects may also employ a shadow repository where they maintain a clone of their main repository and lock this down to only trusted maintainers. 
+
+Finally, when you do publish a security fix, you should make sure to publish the vulnerability as a CVE. Mitre runs the [CVE request process](https://cve.mitre.org/cve/request_id.html). If your project is covered by a particular numbering authority, you should work with them to obtain a CVE number for your vulnerability. If you are not, then you may leverage Mitre as your numbering authority. 
+
+## See also
+* [HackerOne's security policy builder](https://hackerone.com/policy-builder/)
+* [Securitytxt.org](https://securitytxt.org)
+* [NTIA Coordinated Vulnerability Disclosure template] (https://www.ntia.doc.gov/files/ntia/publications/ntia_vuln_disclosure_early_stage_template.pdf)
+
+

From c00d43fee212bc08fa75f88de78d09a91f4e2b7f Mon Sep 17 00:00:00 2001
From: Justin Hutchings <jhutchings1@users.noreply.github.com>
Date: Thu, 22 Aug 2019 21:53:14 -0700
Subject: [PATCH 4/5] Added more metadata to security page

---
 _articles/think-about-security.md | 10 +++++++++-
 script/ebooks                     |  1 +
 2 files changed, 10 insertions(+), 1 deletion(-)

diff --git a/_articles/think-about-security.md b/_articles/think-about-security.md
index f51fa74579..73e4c8dca8 100644
--- a/_articles/think-about-security.md
+++ b/_articles/think-about-security.md
@@ -3,7 +3,15 @@ lang: en
 title: Think about security
 description: Build your open source project with security in mind
 class: best-practices
-image: /assets/images/cards/best-practices.png
+toc: 
+  why-is-security-such-a-big-deal: "Why is security such a big deal"
+  secure-your-software-supply-chain: "Secure your software supply chain"
+  automate-repetitive-tasks: "Automate repetitive tasks"
+  create-a-security-policy: "Create a security policy"
+  responsibly-disclose-your-vulnerabilities: "Responsibly disclose your vulnerabilities"
+order: 11
+related:
+  - best-practices
 ---
 
 ## Why is security such a big deal? 
diff --git a/script/ebooks b/script/ebooks
index b9372a28de..1ecf6d9076 100755
--- a/script/ebooks
+++ b/script/ebooks
@@ -34,6 +34,7 @@ buildBooks([
   '_articles/code-of-conduct.md',
   '_articles/metrics.md',
   '_articles/legal.md',
+  '_articles/think-about-security.md',
 ]);
 
 function buildBooks(files) {

From b438bb3042d0ac21616e82b854012e4836220820 Mon Sep 17 00:00:00 2001
From: Justin Hutchings <jhutchings1@users.noreply.github.com>
Date: Thu, 22 Aug 2019 22:06:14 -0700
Subject: [PATCH 5/5] Clean up formatting on security page

---
 _articles/think-about-security.md | 18 +++++++++++++-----
 1 file changed, 13 insertions(+), 5 deletions(-)

diff --git a/_articles/think-about-security.md b/_articles/think-about-security.md
index 73e4c8dca8..195d20bdda 100644
--- a/_articles/think-about-security.md
+++ b/_articles/think-about-security.md
@@ -16,17 +16,23 @@ related:
 
 ## Why is security such a big deal? 
 
-Even small projects in open-source can have a big impact because of the way ideas move through the community. When it comes to software security, vulnerabilities can move through the community just as quickly as great ideas, so it's important that every open-source project think about how they will approach security. 
+Even small projects in open-source can have a big impact because of the way ideas move through the community. 
+
+When it comes to software security, vulnerabilities can move through the community just as quickly as great ideas! If you've created a library, anyone that takes a dependency on your code has included it in their risk surface for the project they run on top of you. And if you're releasing an app or service, you could be putting user data at risk if you don't handle security vulnerabilities. Either way, it's important that every open-source project maintainer to think about how they will approach security up-front.
 
 ## Secure your software supply chain
 
-Software component analysis is an important part of protecting your software supply chain. When open-source modules that you depend on have vulnerabilities, your users are at risk! Services like [GitHub](https://github.com/), [Snyk](https://snyk.io/), or [WhiteSource](https://www.whitesourcesoftware.com/) offer vulnerability scanning for free for open-source projects. These services will send you security alerts when your projects have a vulnerability and will include information on the impact of the vulnerability.
+Software component analysis is an important part of protecting your software supply chain. 
+
+When open-source modules that you depend on have vulnerabilities, your users are at risk! Services like [GitHub](https://github.com/), [Snyk](https://snyk.io/), or [WhiteSource](https://www.whitesourcesoftware.com/) offer vulnerability scanning for free for open-source projects. These services will send you security alerts when your projects have a vulnerability and will include information on the impact of the vulnerability.
 
 Docker containers are another major source of vulnerabilities. Any of the code that's part of a Docker container that you run could have vulnerabilities. The open-source project [Clair](https://github.com/coreos/clair) does scanning of Docker containers for known vulnerabilities. There are also many commercial security vendors which may perform more comprehensive scanning of Docker containers. 
 
 ## Automate repetitive tasks
 
-Updating dependencies can be a frequent, and repetitive task. Many services offer free dependency updates including [Dependabot](https://dependabot.com/) (which is built-into GitHub as a beta right now), and [GreenKeeper](https://greenkeeper.io/). These tools will automatically generate pull requests with the required version updates in response to security vulnerabilities, and dramatically improve the experience of updating dependencies. 
+Updating dependencies can be a frequent, and repetitive task.
+
+Many services offer free dependency updates including [Dependabot](https://dependabot.com/) (which is built-into GitHub as a beta right now), and [GreenKeeper](https://greenkeeper.io/). These tools will automatically generate pull requests with the required version updates in response to security vulnerabilities, and dramatically improve the experience of updating dependencies. 
 
 ## Create a security policy
 
@@ -48,7 +54,9 @@ Safe harbor provisions are important in terms of setting boundaries as to what c
 
 ## Responsibly disclose your vulnerabilities
 
-When you learn about a security vulnerability in your code, it's important that you fix and disclose that issue responsibly. In open-source projects, this can be challenging, since everything is open by default. 
+When you learn about a security vulnerability in your code, it's important that you fix and disclose that issue responsibly. 
+
+In open-source projects, this can be challenging, since everything is open by default. 
 
 GitHub offers a beta feature called [maintainer security advisories](https://help.github.com/en/articles/about-maintainer-security-advisories) which allow you to discuss, fix, and disclose vulnerabilities without disclosing them before they are ready. Some projects may also employ a shadow repository where they maintain a clone of their main repository and lock this down to only trusted maintainers. 
 
@@ -57,6 +65,6 @@ Finally, when you do publish a security fix, you should make sure to publish the
 ## See also
 * [HackerOne's security policy builder](https://hackerone.com/policy-builder/)
 * [Securitytxt.org](https://securitytxt.org)
-* [NTIA Coordinated Vulnerability Disclosure template] (https://www.ntia.doc.gov/files/ntia/publications/ntia_vuln_disclosure_early_stage_template.pdf)
+* [NTIA Coordinated Vulnerability Disclosure template](https://www.ntia.doc.gov/files/ntia/publications/ntia_vuln_disclosure_early_stage_template.pdf)

**Comment by suusha1340** [comment]
diff --git a/_articles/think-about-security.md b/_articles/think-about-security.md
new file mode 100644
index 0000000000..195d20bdda
--- /dev/null
+++ b/_articles/think-about-security.md
@@ -0,0 +1,70 @@
+---
+lang: en
+title: Think about security
+description: Build your open source project with security in mind
+class: best-practices
+toc: 
+  why-is-security-such-a-big-deal: "Why is security such a big deal"
+  secure-your-software-supply-chain: "Secure your software supply chain"
+  automate-repetitive-tasks: "Automate repetitive tasks"
+  create-a-security-policy: "Create a security policy"
+  responsibly-disclose-your-vulnerabilities: "Responsibly disclose your vulnerabilities"
+order: 11
+related:
+  - best-practices
+---
+
+## Why is security such a big deal? 
+
+Even small projects in open-source can have a big impact because of the way ideas move through the community. 
+
+When it comes to software security, vulnerabilities can move through the community just as quickly as great ideas! If you've created a library, anyone that takes a dependency on your code has included it in their risk surface for the project they run on top of you. And if you're releasing an app or service, you could be putting user data at risk if you don't handle security vulnerabilities. Either way, it's important that every open-source project maintainer to think about how they will approach security up-front.
+
+## Secure your software supply chain
+
+Software component analysis is an important part of protecting your software supply chain. 
+
+When open-source modules that you depend on have vulnerabilities, your users are at risk! Services like [GitHub](https://github.com/), [Snyk](https://snyk.io/), or [WhiteSource](https://www.whitesourcesoftware.com/) offer vulnerability scanning for free for open-source projects. These services will send you security alerts when your projects have a vulnerability and will include information on the impact of the vulnerability.
+
+Docker containers are another major source of vulnerabilities. Any of the code that's part of a Docker container that you run could have vulnerabilities. The open-source project [Clair](https://github.com/coreos/clair) does scanning of Docker containers for known vulnerabilities. There are also many commercial security vendors which may perform more comprehensive scanning of Docker containers. 
+
+## Automate repetitive tasks
+
+Updating dependencies can be a frequent, and repetitive task.
+
+Many services offer free dependency updates including [Dependabot](https://dependabot.com/) (which is built-into GitHub as a beta right now), and [GreenKeeper](https://greenkeeper.io/). These tools will automatically generate pull requests with the required version updates in response to security vulnerabilities, and dramatically improve the experience of updating dependencies. 
+
+## Create a security policy
+
+ A security policy helps your community by a) letting them know what you will support in terms of security issues, and b) letting them know how to work with you when they discover security issues.  
+
+### Scope/Supported versions
+
+Tell users about the parts of your project which are within scope of the security policy. For example, if you will not accept security bugs on test collateral, or samples, you should describe those limitations here. If your project has any official releases or versions, you should let users know which ones are being officially supported here. 
+
+### Disclosure policy 
+
+In the disclosure policy section, describe how and where users should disclose security vulnerabilities here. This is the right place for a link to the private inbox you might host, a link to your HackerOne, or a link to your own custom website. 
+
+If you have any security requirements on the communications (eg, no attachments, signed with a PGP key, etc), you should also list them here. 
+
+### Safe harbor 
+
+Safe harbor provisions are important in terms of setting boundaries as to what constitutes ethical hacking, vs unlawful hacking on your project. These sections are important, so if your company or project has lawyers, now's the time to call them. A good security policy should encourage security researchers to report security issues responsibly, and a good safe harbor provision gives those researches confidence that they won't be prosecuted for doing so. The [GitHub Bug Bounty Program](https://github.blog/2019-02-19-five-years-of-the-github-bug-bounty-program/#legal-safe-harbor) provides more information about how GitHub uses a safe harbor provision to protect hackers provided that they abide by certain limitations. 
+
+## Responsibly disclose your vulnerabilities
+
+When you learn about a security vulnerability in your code, it's important that you fix and disclose that issue responsibly. 
+
+In open-source projects, this can be challenging, since everything is open by default. 
+
+GitHub offers a beta feature called [maintainer security advisories](https://help.github.com/en/articles/about-maintainer-security-advisories) which allow you to discuss, fix, and disclose vulnerabilities without disclosing them before they are ready. Some projects may also employ a shadow repository where they maintain a clone of their main repository and lock this down to only trusted maintainers. 
+
+Finally, when you do publish a security fix, you should make sure to publish the vulnerability as a CVE. Mitre runs the [CVE request process](https://cve.mitre.org/cve/request_id.html). If your project is covered by a particular numbering authority, you should work with them to obtain a CVE number for your vulnerability. If you are not, then you may leverage Mitre as your numbering authority. 
+
+## See also
+* [HackerOne's security policy builder](https://hackerone.com/policy-builder/)
+* [Securitytxt.org](https://securitytxt.org)
+* [NTIA Coordinated Vulnerability Disclosure template](https://www.ntia.doc.gov/files/ntia/publications/ntia_vuln_disclosure_early_stage_template.pdf)
+
+
diff --git a/script/ebooks b/script/ebooks
index b9372a28de..1ecf6d9076 100755
--- a/script/ebooks
+++ b/script/ebooks
@@ -34,6 +34,7 @@ buildBooks([
   '_articles/code-of-conduct.md',
   '_articles/metrics.md',
   '_articles/legal.md',
+  '_articles/think-about-security.md',
 ]);
 
 function buildBooks(files) {

**Pull request closed without merging**